sample/CustomHandlerRetry/package.json

```
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "express": "^4.17.2"
```

```suggestion
    "express": "4.17.2"
```
nit: pin version to avoid surprised in future

---

The existing code should be fine - pinning is handled by the package-lock.json file which is already included in this PR (as it should be)